ofctrl/ofctrl_test.go

```diff
@@ -319,100 +266,65 @@ func TestCreateDeleteFlow(t *testing.T) {
 		TcpFlags:     &tcpFlag,
 		TcpFlagsMask: &tcpFlag,
 	})
-	if err != nil {
-		t.Errorf("Error creating tcp flow. Err: %v", err)
-	}
-
-	log.Infof("Creating tcp flow: %+v", tcpFlow)
+	assert.Nil(t, err, fmt.Sprintf("Error creating tcp flow: %v", err))
```

Comment:
you do not need `fmt.Sprintf`, you can use the `assert.Nilf` function when needed

in this case, you should also be using `assert.NoError(t, err, "Error creating tcp flow")`. There is no need to print the error in the message, as the library will do it for you.

Reply:
Updated.